File: app/dbmodels/NetworkName.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'NetworkName',
    'menu'          => ''
);
$table['fields'] = array(
    'id'               => DBModels::type('increments',
                                         array('visible' => false)),
    'item_type'        => DBModels::type('string',
                                        array('visible' => false,
                                              'size'    => 100)),
    'item_id'          => DBModels::type('integer',
                                         array('visible' => false)),
    'name'             => DBModels::type('string'),
    'is_deleted'       => DBModels::type('boolean',
                                         array('visible' => false)),
    'is_dynamic'       => DBModels::type('boolean',
                                         array('visible' => false)),
    'comment'          => DBModels::type('text'),
    'fqdn_id'         => DBModels::type('integer'),
);

$table['relationships']['fqdn'] = array(
        'type'      => 'belongsTo',
        'item'      => 'Fqdn');
$table['relationships']['item'] = array(
        'type'      => 'morphTo');

include "_commonentitylink.php";
include "_commonrecursiveentitylink.php";

File: fusionglpi/database/networkname.py
from fusionglpi import db

class Networkname(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    entity_id = db.Column(db.Integer, db.ForeignKey('entity.id'))
    is_recursive = db.Column(db.Boolean, nullable=False, server_default="0")
    item_type = db.Column(db.String(100, None, True))
    item_id = db.Column(db.Integer, nullable=False, server_default="0")
    name = db.Column(db.String(255, None, True))
    is_deleted = db.Column(db.Boolean, nullable=False, server_default="0")
    is_dynamic = db.Column(db.Boolean, nullable=False, server_default="0")
    comment = db.Column(db.Text)
    fqdn_id = db.Column(db.Integer, db.ForeignKey('fqdn.id'))
